### PR TITLE
Add noetic-alpine3.17

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

File: backports/py3-qt5.v3.14/ENABLE_ON [hunk 1]
```diff
@@ -0,0 +1 @@
+v3.11
```
Comment (f-fl0) [inline backports/py3-qt5.v3.14/ENABLE_ON:1]: Should we also add `v3.14`?

File: build-repo.sh [hunk 2]
```diff
@@ -51,10 +55,14 @@ echo
 if [ ! -f ${PACKAGER_PRIVKEY} ]; then
   echo "======== WARN: PACKAGER_PRIVKEY is not present ======="
   abuild-keygen -a -i -n
-  sudo cp ${HOME}/.abuild/*.pub /etc/apk/keys/
   RESIGN=true
+else
+  echo "Using ${PACKAGER_PRIVKEY}"
+  openssl rsa -in ${PACKAGER_PRIVKEY} -pubout | sudo tee ${PACKAGER_PRIVKEY}.pub
 fi
```
Comment (f-fl0) [inline build-repo.sh:62]: Out of curiosity, why was this not necessary before?

**Review by f-fl0 — APPROVED** [review]
LGTM. I didn't check in details the ROS packages though. I left some optional comments.

File: build-repo.sh (hunk 2 above)
Comment (at-wat) [reply build-repo.sh:62]: I guess it depends on the abuild version.

File: backports/py3-qt5.v3.14/ENABLE_ON (hunk 1 above)
Comment (at-wat) [reply backports/py3-qt5.v3.14/ENABLE_ON:1]: On 3.14, py3-qt5 in official Alpine repository can be used. I'll fix the directory name

**Review by Taka-Kazu — APPROVED** [review]

**Pull request merged**